src/js/constants/NameMappings.js

```diff
@@ -0,0 +1,13 @@
+var AppStatus = require("../constants/AppStatus");
+
+const NameMappings = {
+  statusNameMapping: {
```

Comment:
Ok, looked through the constant files.
I couldn't find any other file that fits into that scheme.

@philipnrmn What do you mean with "label" ?